js/ui/widget/ui.widget.js

```diff
@@ -1,30 +1,25 @@
-var $ = require("../../core/renderer"),
-    eventsEngine = require("../../events/core/events_engine"),
-    events = require("../../events/"),
-    Action = require("../../core/action"),
-    extend = require("../../core/utils/extend").extend,
-    inArray = require("../../core/utils/array").inArray,
-    each = require("../../core/utils/iterator").each,
-    commonUtils = require("../../core/utils/common"),
-    typeUtils = require("../../core/utils/type"),
-    DOMComponentWithTemplate = require("../../core/dom_component_with_template"),
-    selectors = require("./selectors"),
-    eventUtils = require("../../events/utils");
-
-require("../../events/click");
-require("../../events/hover");
-require("../../events/core/emitter.feedback");
-
-const { hover, focus, active, dxClick, keyboard } = events;
-
-var WIDGET_CLASS = "dx-widget",
-    ACTIVE_STATE_CLASS = "dx-state-active",
-    DISABLED_STATE_CLASS = "dx-state-disabled",
-    INVISIBLE_STATE_CLASS = "dx-state-invisible",
-    HOVER_STATE_CLASS = "dx-state-hover",
-    FOCUSED_STATE_CLASS = "dx-state-focused",
-    FEEDBACK_SHOW_TIMEOUT = 30,
-    FEEDBACK_HIDE_TIMEOUT = 400;
+import $ from '../../core/renderer';
+import Action from '../../core/action';
+import DOMComponentWithTemplate from '../../core/dom_component_with_template';
+import { active, dxClick, focus, hover, keyboard } from '../../events/';
+import { deferRender, deferRenderer, noop } from '../../core/utils/common';
+import { each } from '../../core/utils/iterator';
+import { extend } from '../../core/utils/extend';
+import { focusable as focusableSelector } from './selectors';
+import { inArray } from '../../core/utils/array';
+import { isFakeClickEvent } from '../../events/utils';
+import { isPlainObject, isDefined } from '../../core/utils/type';
+
+import '../../events/click';
+import '../../events/core/emitter.feedback';
+import '../../events/hover';
+
+function setAttribute(name, value, target) {
```

Comment:
Should we use an arrow function?
`const setAttribute = (name, value, target) => {...}`